src/app/pages/OpenWalletPage/index.tsx

```diff
@@ -65,6 +65,15 @@ export function SelectOpenMethod() {
           />
         )}
       </Box>
+
+      <Box direction="row-responsive" justify="start" margin={{ top: 'medium' }} gap="medium">
+        <a href="https://docs.oasis.dev/general/manage-tokens/oasis-wallets#how-can-i-transfer-rose-tokens-from-my-bitpie-wallet-to-my-oasis-wallet">
+          {t(
+            'openWallet.bitpie.warning',
+            '❗ BitPie wallet users: You cannot import the mnemonic phrase or the private key directly from your BitPie wallet. Check documentation for details.',
```

Comment:
where should they look?

Reply:
add a link or remove that last sentence

Reply:
@pro-wh the whole thing is a link.

We could give more control to translators with
```jsx
      <Trans
        i18nKey="openWallet.bitpie.warning"
        t={t}
        defaults="❗ BitPie wallet users: You cannot import the mnemonic phrase or the private key directly from your BitPie wallet. Check <0>documentation for details.</0>"
        components={[
          <Anchor
            href="https://docs.oasis.dev/general/manage-tokens/oasis-wallets#how-can-i-transfer-rose-tokens-from-my-bitpie-wallet-to-my-oasis-wallet"
          />,
        ]}
      />
```

Reply:
Thanks @lukaw3d . I knew there had to be a react way to put items inside translation terms. :)

Applied.

Reply:
oh thanks. man, link styling these days :laughing: